### PR TITLE
Update translation whenever the app resumes (AIC-67)

### PR DESCRIPTION
**Pull request by sam33rdhakal**
Updates:
* Updates string resources with correct translations
* Updates the activity's content when app locale is updated
* Replaces hardcoded strings with the localized translations

### CODE REVIEW

File: info/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <string name="chooseYourLanguage">请选择您的首选语言</string>
+    <string name="language_settings">语言设置</string>
+    <string name="become_a_member">成为会员</string>
+    <string name="enjoy_free_year_long_admission">享受免费的年票。</string>
+    <string name="join_now">现在加入</string>
+    <string name="museum_information">博物馆信息</string>
+    <string name="location_settings">地点设置</string>
+    <string name="support_by">技术支持</string>
+    <string name="buyTickets">购票</string>
+    <string name="versionInfo" translatable="false">" 版本 %1$s"</string>
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply info/src/main/res/values-zh/strings.xml]: Updated.

File: info/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <string name="language_settings">Configuración de idiomas</string>
+    <string name="become_a_member">Hágase miembro</string>
+    <string name="enjoy_free_year_long_admission">Disfrute de admisión gratuita durante todo el año.</string>
+    <string name="join_now">Unirse ahora mismo</string>
+    <string name="museum_information">Información sobre el museo</string>
+    <string name="location_settings">Configuración de ubicación</string>
+    <string name="support_by">Patrocinado por</string>
+    <string name="buyTickets">Comprar boletos</string>
+    <string name="signIn">Iniciar sesión</string>
+    <string name="versionInfo" translatable="false">Versión %1$s</string>
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply info/src/main/res/values-es/strings.xml]: Updated.

File: search/src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="gallery">Galería %s</string>
+    <string name="artworks">Obras de arte</string>
+    <string name="couldnt_find_top_text">Lo sentimos, no encontramos resultados</string>
+    <string name="couldnt_find_bottom_text">Para obtener ayuda, visite al representante del museo más cercano o [visite nuestro sitio web].</string>
```
[the remark this answers is not in the document]
Comment (Cliabhach) [reply search/src/main/res/values-es/strings.xml]: Huh. Don't worry about these now; we'll answer this question later on.

File: info/src/main/res/values-es/strings.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="chooseYourLanguage">Elija su idioma de preferencia</string>
+    <string name="language_settings">Configuración de idiomas</string>
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply info/src/main/res/values-es/strings.xml]: Updated

File: base/src/main/res/values-es/strings.xml [hunk 5]
```diff
@@ -1,4 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="name_of_this_language">Español</string>
+    <string name="welcome">Bienvenido</string>
+    <string name="welcomePrompt">Explore el museo con su guía de bolsillo personalizada.Si es miembro, regístrese para obtener un acceso mejorado.</string>
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply base/src/main/res/values-es/strings.xml]: You are right. Updated the view model to get welcomePrompt from `ArticGeneralInfo `'s `homeMemberPromptText`

File: details/src/main/kotlin/edu/artic/tours/TourDetailsViewModel.kt [hunk 6]
```diff
@@ -72,7 +73,7 @@ class TourDetailsViewModel @Inject constructor(
 
         //TODO: replace Stops with localized string when localizer is created
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply details/src/main/kotlin/edu/artic/tours/TourDetailsViewModel.kt]: Done.

File: location/src/main/res/values-en/strings.xml [hunk 7]
```diff
@@ -3,4 +3,10 @@
     <string name="location_title">Location Settings</string>
     <string name="locationSettingsHeader">Where Are You?</string>
     <string name="locationSettingsBody">We’d like to use your location to help you navigate the museum.</string>
+    <string name="locationSettingsLocationDenied">Location Disabled</string>
+    <string name="locationSettingsLocationAllowed">Location Enabled</string>
+    <string name="locationPromptNotNow">Now Right Now</string>
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply location/src/main/res/values-en/strings.xml]: Updated.

File: map/src/main/kotlin/edu/artic/map/SearchObjectDetailsFragment.kt [hunk 8]
```diff
@@ -33,8 +33,7 @@ class SearchObjectDetailsFragment : BaseViewModelFragment<SearchObjectDetailsVie
     override val viewModelClass: KClass<SearchObjectDetailsViewModel>
         get() = SearchObjectDetailsViewModel::class
 
-    override val title: String
-        get() = "Artworks"
+    override val title = R.string.noTitle
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply map/src/main/kotlin/edu/artic/map/SearchObjectDetailsFragment.kt]: My mistake. Updated. Thanks

File: info/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -16,25 +16,26 @@
...
     <string name="signIn">Sign In</string>
-    <string name="homeZipCode">Home Zip Code</string>
+    <string name="homeZipCode">Zip Code</string>
     <string name="memberId">Member ID</string>
     <string name="changeInformation">Change Information</string>
     <string name="switchCardHolder">Switch Cardholder</string>
     <string name="memberCard">Member Card</string>
     <string name="errorDialogTitle">Error</string>
-    <string name="ok">OK</string>
+    <string name="ok" translatable="false">@android:string/ok</string>
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply info/src/main/res/values/strings.xml]: Yes, it is working properly.

File: location/src/main/res/values-en/strings.xml [hunk 10]
```diff
@@ -3,4 +3,10 @@
...
+    <string name="locationPromptOk">OK, Use My Location</string>
+    <string name="locationSettingsLocationServiceOff">Location Service Off</string>
+    <string name="locationSettingsLocationNeverAsked">Location Service Off</string>
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply location/src/main/res/values-en/strings.xml]: @raigex could you answer this one, please.

File: info/src/main/res/values/strings.xml (hunk 9 above)
[the remark this answers is not in the document]
Comment (Cliabhach) [reply info/src/main/res/values/strings.xml]: Excellent, that's great news.

**Comment by Cliabhach** [comment]
Great stuff. I'll merge it in when I get approval from Circle CI.

**Pull request merged**